src/gravel/tests/unit/controllers/test_config.py

```diff
@@ -1,19 +1,11 @@
-from datetime import datetime
 from pathlib import Path
 
-from gravel.controllers.config \
-    import Config, DeploymentStage
+from gravel.controllers.config import Config
 
 
 def test_config_version(fs):
     config = Config()
-    assert config.config.version == 3
-
-
-def test_deployment_state(fs):
```

Comment:
We need to open a ticket to re-add this and the disabled_test_services if this gets merged.

Reply:
this was removed because `deployment_state` no longer exists. We might want to test `node_state`, but unless it's through a functional test I'm not sure it's really worth it.